components/cart/CartContentDetail.js

```diff
@@ -0,0 +1,118 @@
+import { useRouter } from 'next/router';
+import styled from '@emotion/styled';
+import { DELETE } from '@apis/defaultApi';
+import { ThemeGray1 } from '@utils/constants/themeColor';
+import CartProduct from '@components/cart/CartProduct';
+
+export default function CartContentDetail({
+  userId,
+  cartItemDtoList,
+  checkBoxStates,
+  cartCheckBoxChange,
+  cartInfoChange,
+}) {
+  const router = useRouter();
+
+  const ProductCheckSection = styled.div`
+    position: relative;
+    display: block;
+    flex: 0 0 auto;
+    margin-right: 5px;
+    padding: 5px;
+    width: 20px;
+  `;
+
+  const ProductDetailSection = styled.div`
+    position: relative;
+    display: flex;
+    flex: 1 0 auto;
+
+    @media (max-width: 1024px) {
+      flex-direction: column;
+      width: min-content;
+    }
+    @media (min-width: 1024px) {
+      flex-direction: row;
+    }
+  `;
+
+  const ProductDeleteSection = styled.div`
+    position: relative;
+    flex: 0 0 auto;
+    padding-left: 5px;
+    display: block;
+    margin-left: auto;
+    width: 20px;
+
+    &:hover {
+      cursor: pointer;
+    }
+  `;
+
+  const CartDetailSection = styled.div`
+    padding-top: 5px;
+    border-top: 1px solid white;
+    border-bottom: 1px solid white;
+    display: flex;
+  `;
+
+  const deleteCartItem = ({ input }) => {
+    const CartItemDeleteRequestDto = {
+      userId: userId,
+      cartItemId: input,
+    };
+    if (confirm('장바구니에서 삭제하시겠습니까?')) {
+      DELETE(`/cart/delete`, CartItemDeleteRequestDto);
+      router.reload();
```

Comment:
```suggestion
const [cartItemList, setCartItemList] = useState(cartItemDtoList);
.....
      const arrIdx = cartItemList.findIndex((e) => e.cartItemId == input);
      let copyArr = [...cartItemList];
      if (arrIdx > -1) copyArr.splice(arrIdx, 1);

      setCartItemList(copyArr);
```
이렇게 하면 어때용 ? 그럼 리로드 안하고 부드럽게 삭제할 수 있어요!

Reply:
삭제할때 저것만 삭제하면 되는게 아니고 바꿔줘야할게 많아서 부드럽게 삭제는 다른기능을 우선적으로 적용하고 후순위로 좀 미뤄둬야할거 같아요 ㅠㅠ

Reply:
흐규 넵!!